common/static/common/templates/components/search-field.underscore

```diff
@@ -1,9 +1,9 @@
-<div class="page-header-search wrapper-search-<%= type %>">
+<div class="page-header-search wrapper-search-<%= HtmlUtils.ensureHtml(type) %>">
```

Comment:
I would change all of these usages of ``ensureHtml`` to just use ``<%-`` unless the variable really does contain HTML.